### PR TITLE
Getting started using XKS

### PR DESCRIPTION
**Pull request by NissesSenap**

### CODE REVIEW

File: docs/services/xks/operator-guide.md [hunk 1]
```diff
@@ -3,6 +3,239 @@ id: operator-guide
...
+### Setup XKS
+
+In this case we will only setup a single XKS cluster in one environment, in our case dev.
+It's easy to add more environments when you have created your first one.
+
+At Xenit we are using terraform modules that we share [upstream](https://github.com/XenitAB/terraform-modules)
+
+To setup XKS we will utalize 3 modules:
+
+- governance-global
```
Comment (simongottschlag) [inline docs/services/xks/operator-guide.md:36]: regional?

File: docs/services/xks/operator-guide.md [hunk 2]
```diff
@@ -3,6 +3,239 @@ id: operator-guide
...
+
+### Add PAT to Azure Key Vaults
+
+To make it possible for terraform to reach the PAT in a easy and secure way we have chosen to store the PAT in Azure Key Vaults which you need to add manually.
+
+Now you need access to the azure portal.
+
+In azure portal search for "Key vaults" and pick the core one that matches the unique_suffix that you have specified in global.tfvars, in our case 1234.
+
+Key vaults -> core-1234 -> Secrets -> Generate/Import
```
Comment (simongottschlag) [inline docs/services/xks/operator-guide.md:176]: cli example as well

File: docs/services/xks/operator-guide.md [hunk 3]
```diff
@@ -3,6 +3,239 @@ id: operator-guide
...
+
+All of this is configured under aks/variables/prod.tfvars
+
+```prod.tfvars
+environment = "dev"
+dns_zone    = "dev.aks.xenit.io"
+
+aks_config = {
+  kubernetes_version = "1.19.7"
+  sku_tier           = "Free"
```
Comment (simongottschlag) [inline docs/services/xks/operator-guide.md:228]: @phillebaba should the we use Standard (paid with SLA) by default?

**Review by simongottschlag — COMMENTED** [review]
Some comments

File: docs/services/xks/operator-guide.md (hunk 1 above)
Comment (NissesSenap) [reply docs/services/xks/operator-guide.md:36]: Yeah I thought of that as well.
Do we have some word for having 2 modules in 1 folder?
groups or something. My thught was that I wanted to have a easy connection to the developers file strucutre. 
But I should probably just go with 4 modules and explain how the folder strucutre looks like.

File: docs/services/xks/operator-guide.md (hunk 2 above)
Comment (NissesSenap) [reply docs/services/xks/operator-guide.md:193]: Please review the cli commands, it works at least :)

**Comment by NissesSenap** [comment]
I have totally forgotten about this.
What do you say, should we merge?
It will need to be continuously updated but I would get this initial merge done.

**Pull request merged**